packages/template-retail-react-app/app/utils/utils.js

```diff
@@ -292,3 +292,34 @@ export const resolveLocaleFromUrl = (url) => {
         (locale) => locale.alias === defaultLocale || locale.id === defaultLocale
     )
 }
+
+const toCamel = (str) => {
+    if (str.startsWith('_') || str.startsWith('c_')) {
+        return str
+    }
+    return str.replace(/([-_][a-z])/gi, ($1) => {
+        return $1.toUpperCase().replace('-', '').replace('_', '')
+    })
+}
+
+const isObject = (obj) => {
+    return obj === Object(obj) && !Array.isArray(obj) && typeof obj !== 'function'
+}
+
+export const keysToCamel = (obj) => {
+    if (isObject(obj)) {
+        const n = {}
+
+        Object.keys(obj).forEach((k) => {
+            n[toCamel(k)] = keysToCamel(obj[k])
+        })
+
+        return n
+    } else if (Array.isArray(obj)) {
+        return obj.map((i) => {
+            return keysToCamel(i)
+        })
```

Comment:
```suggestion
        return obj.map(keysToCamel)
```
Mostly just save two lines, but also _ewww_, why are you iterating an array and using `i` for the value, rather than index? 😝